src/context.h

```diff
@@ -70,9 +69,7 @@ namespace librealsense
 
     enum class backend_type
     {
-        standard,
-        record,
-        playback
+        standard
```

Comment:
do we need to let this enum, now that it has only one unique value?

Reply:
I started removing it, but kept it because of changed in the `dds` branch.

Reply:
you mean there is another value in this enum for the dds branch?

Reply:
Not another value -- other constructors, one with no parameters and another one, with another parameter type.

Reply:
ok